app/scripts/arbeitseinsaetze/list/arbeitseinsaetzelist.html

```diff
@@ -34,8 +34,19 @@
                 </td>
                 <td>
                   <div class="btn-group">
-                    <a role="button" class="btn btn-sm btn-warning" ng-click="quit(arbeitseinsatz)"><span class="glyphicon glyphicon-minus"></span></a>
-                    <a role="button" class="btn btn-sm btn-info" ng-click="edit(arbeitseinsatz)"><span class="glyphicon glyphicon-edit"></span></a>
+                    <script type="text/ng-template" id="workAssignmentInfoTooltipTemplate.html">
+                      <span translate> 
+                        Sie können diese Zuordnung nicht entfernen, da sie zu nahe am Ausführungsdatum liegt
+                      </span>
+                    </script>
+                    <div ng-if="(arbeitseinsatz.zeitBis.getTime()<limitDateForDeletion)">
+                      <a ng-disabled="(arbeitseinsatz.zeitBis.getTime()<limitDateForDeletion)" role="button" uib-tooltip-template="'workAssignmentInfoTooltipTemplate.html'" class="btn btn-sm btn-warning ng-scope" ng-click="(arbeitseinsatz.zeitBis.getTime()<limitDateForDeletion) || quit(arbeitseinsatz)"><span class="glyphicon glyphicon-minus"></span></a>
```

Comment:
I still don't see the need for the ng-disabled here. It will always be disabled, no? Since it's the same statement as in the ng-if.
Sorry for being picky.